py-polars/tests/unit/test_errors.py

```diff
@@ -286,7 +286,7 @@ def test_lazy_concat_err() -> None:
     for how in ["horizontal"]:
         with pytest.raises(
             ValueError,
-            match="'LazyFrame' only allows {{'vertical', 'diagonal'}} concat strategy.",
+            match="'LazyFrame' .* allows {'vertical', 'diagonal','align'} .* strategies.",
```

Comment:
Is replacing `only` and `concat` with  `.*` intentional? If so, why?

Reply:
Depending on your lint settings it would complain about a long line with the new param added; I can make it a two-line string continuation instead, and keep the whole message :)

Reply:
Oh, I now see it is the regex of the test.

I thought it was a docstring. Nevermind.